src/toolbar/index.ts

```diff
@@ -81,6 +81,10 @@ export class Toolbar extends I18nMixin(ThemedMixin(WidgetBase))<ToolbarPropertie
 			classes,
 			heading
 		} = this.properties;
+		const actions = v('div', {
+			classes: this.theme(this._collapsed ? css.slidePaneActions : css.actions),
```

Comment:
I think that we can always use `css.actions` here. We can differentiate between slide pane / toolbar placement via the `collapsed` class. This makes the dom / classes more predictable for the end user.

Reply:
Thanks, this is definitely a better solution! Updated